kolibri/content/models.py

```diff
@@ -158,14 +160,15 @@ def get_descendant_kind_counts(self):
 
 @python_2_unicode_compatible
 class Language(ContentDatabaseModel):
-    id = models.CharField(max_length=7, primary_key=True)
+    id = models.CharField(max_length=14, primary_key=True)
     lang_code = models.CharField(max_length=3, db_index=True)
-    lang_subcode = models.CharField(max_length=3, db_index=True, blank=True, null=True)
-
-    objects = ContentQuerySet.as_manager()
```

Comment:
Ugh - I crossed the streams... it soon won't be, but I should reinstate it here.